velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1096,6 +1096,28 @@ struct ParseDateTimeFunction {
   }
 };
 
+template <typename T>
+struct CurrentTimeStampFunction : public TimestampWithTimezoneSupport<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  int16_t timezoneId;
+
+  FOLLY_ALWAYS_INLINE void initialize(const core::QueryConfig& config) {
+    auto sessionTzName = config.sessionTimezone();
```

Comment:
Nit : The sessionTzName variable is not needed. You can use config.sessionTimezone() directly in if statement below.

Reply:
Thanks, but I may want to keep it there, it's more readable and we use it twice later.

Reply:
Sorry my bad. Didn't realize it was used later.